docs/doc/60-contributing/03-rfcs/20220530-user-friendly-releases.md

```diff
@@ -0,0 +1,71 @@
+---
+title: User-friendly Releases
+description: RFC for user-friendly releases
+---
+
+# Summary
+
+A simple user-friendly release solution.
+
+# Motivation
+
+In the past, only specific binaries were included in the Databend release. Understanding how to operate and maintain Databend often requires extensive discussion.
+
+This proposal aims to improve this process by providing some simple tips and default configs to help make the process painless for users.
+
+# Detailed design
+
+The release directory structure is roughly as follows:
+
+```rust
+.
+├── bin
+│   ├── databend-query
+│   ├── databend-meta
+│   └── databend-metactl (?)
+├── configs
+│   ├── databend-query.toml
+│   └── databend-meta.toml
+├── docs 
+│   ├── basic-guide.md
+│   ├── faq.md
+│   └── mini-tutorial.md (?)
+├── data/ (?)
+├── scripts/ (?)
+│   ├── bootstrap.sh
+│   └── benchmark.sh
+├── Makefile (?)
```

Comment:
Users could be confused to have a `Makefile` in releases.

Reply:
I agree with you, I will remove it.